ERCS/erc-7638.md

```diff
@@ -0,0 +1,190 @@
+---
+eip: 7638
+title: Optimized Calls Encoding
```

Comment:
Your title is a bit too general. Try to point out something specific about your proposal, and give an idea of where in the stack (contracts, RPC endpoints, etc.) that your proposal is touching.